### PR TITLE
Uninstall module used on PS 1.6 before using this one

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This PR is a cherry-pick of #10 where PRs that made PrestaShop < 1.7.8.0 incompatible with this module have been reverted.
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Partially PrestaShop/Prestashop#22668
| How to test?  | 1. Be sure that both `blocksearch` & `ps_searchbar` are **not** installed in a 1.7.7.x shop<br>2. Install [this modified version of blocksearch](https://github.com/PrestaShop/ps_searchbar/files/5943485/blocksearch.zip)<br>3. Install `ps_searchbar` with this PR<br>4. Check that `blocksearch` is removed

### CODE REVIEW

File: ps_searchbar.php [hunk 1]
```diff
@@ -36,6 +36,11 @@
 
 class Ps_Searchbar extends Module implements WidgetInterface
 {
+    /**
+     * @var string Name of the module running on PS 1.6.x. Used for data migration.
+     */
+    const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```
Comment (Progi1984) [inline ps_searchbar.php:42]:
```suggestion
    private const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

**Review by Progi1984 — APPROVED** [review]

File: ps_searchbar.php [hunk 2]
```diff
@@ -36,6 +36,11 @@
 
 class Ps_Searchbar extends Module implements WidgetInterface
 {
+    /**
+     * @var string Name of the module running on PS 1.6.x. Used for data migration.
+     */
+    private const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```
Comment (Progi1984) [inline ps_searchbar.php:42]:
```suggestion
    const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```

Oh shit, it's 5.6 compatible. Sorry

**Review by Progi1984 — COMMENTED** [review]
A sorry feedback

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @atomiix !

By following the how-to-test steps, it's OK. After installing ps_searchbar, the search bar is displayed OK and works fine in FO.

QA :heavy_check_mark:

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @atomiix

**Pull request merged**